RELEASE.md

```diff
@@ -176,8 +177,9 @@ Assume `M.m.p` is the new version.
     ```
 
 - Finally, publish the release!
+- The release pipeline will start automatically upon publication and all packages and container images will be uploaded to the stable repositories.
```

Comment:
@LucaGuerra 
How can the release manager get that artifacts have been published? Generally, what should the release manager check before moving forward?

Reply:
Just to make sure I understand, is your suggestion to add the link or instructions (i.e. click on the actions button) to take a look at the github actions pipeline to check in on the progress? In that case I agree!

Reply:
Yes! Link + instructions. :pray:

Reply:
Done!